ui/tslint.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["tslint:recommended", "tslint-react", "tslint-eslint-rules", "tslint-config-prettier"],
```

Comment:
Can you elaborate on this removal?
My current understanding is that we are not enforcing `tslint` in CI and we have a bunch of violations.
But now trying to run in `ui` `npx tslint --project .` results in:
```
Tried to lint <each TSX file> but found no valid, enabled rules for this file type and file path in the resolved configuration.
```

I think we should either completely remove it or start to check the results in CI, wdyt?

Reply:
I removed that because tslint integration in WebStorm was causing all sorts of false positives.  I don't have much experience with tslint and haven't found the time to work on it.  But I do think we should agree on a set of linting rules and apply those rules to all our UI projects.  Enda started doing this here:

https://github.com/Apicurio/apicurio-configs/tree/main/eslint-config

But I haven't picked up the activity yet.

Also checking the results of linting in CI makes sense once we agree on the rules.

Reply:
Agreed, opened:
https://github.com/Apicurio/apicurio-registry/issues/2696

to track this down and unblock this PR.